core/src/test/kotlin/fr/sncf/osrd/standalone_sim/StandaloneSimulationTest.kt

```diff
@@ -355,6 +358,55 @@ class StandaloneSimulationTest {
         assertEquals(expected, safetySpeedRanges)
     }
 
+    /**
+     * Test that the safety speed is not applied to stops in a range of ETCS_LEVEL2 signaling system
+     */
+    @Test
+    fun testSafetySpeedWithETCSLevel2() {
+        // Path length = 10400m
+        // SIGNAL OFFSETS on this path: 150m and 10150m
+        // Then, buffer stop at the end of the last route: 10400m
+        // A ETCS_LEVEL2 signaling range is added between 9500m and 10400m
+        val schedule =
+            listOf(
+                SimulationScheduleItem(
+                    Offset(42.meters),
+                    42.seconds,
+                    42.seconds,
+                    STOP,
+                ),
+                // Stop that will be skipped from safety speed range (in ETCS_LEVEL2 range),
+                // associated to the 10150m signal
+                SimulationScheduleItem(
+                    Offset(10_000.meters),
+                    42.seconds,
+                    42.seconds,
+                    STOP,
```

Comment:
nit: we could make it a short-slip stop to be more exhaustive on tests
```suggestion
                    SHORT_SLIP_STOP,
```

Reply:
Done